src/Misc/layoutbin/powershell/Add-MSBuildCapabilities.ps1

```diff
@@ -15,6 +15,17 @@ $null = Add-CapabilityFromRegistry -Name "MSBuild_3.5" -Hive 'LocalMachine' -Vie
 $null = Add-CapabilityFromRegistry -Name "MSBuild_4.0" -Hive 'LocalMachine' -View 'Registry32' -KeyName $keyName40 -ValueName 'MSBuildToolsPath' -Value ([ref]$latest)
 $null = Add-CapabilityFromRegistry -Name "MSBuild_12.0" -Hive 'LocalMachine' -View 'Registry32' -KeyName $keyName12 -ValueName 'MSBuildToolsPath' -Value ([ref]$latest)
 $null = Add-CapabilityFromRegistry -Name "MSBuild_14.0" -Hive 'LocalMachine' -View 'Registry32' -KeyName $keyName14 -ValueName 'MSBuildToolsPath' -Value ([ref]$latest)
+$vs15 = Get-VisualStudio_15_0
+if ($vs15 -and $vs15.Path) {
+    # Add MSBuild_15.0.
+    # End with "\" for consistency with old MSBuildToolsPath value.
+    $msbuild15 = ([System.IO.Path]::Combine($vs15.Path, 'MSBuild\15.0\Bin')) + '\'
```

Comment:
I think this will suffice for RC2. For RTW I need to follow-up with VS folks to determine whether there is a better way than hardcoding this relative path.